File: app/src/main/java/com/yp/payment/model/OrderDetail.java
package com.yp.payment.model;

public class OrderDetail {
    private Integer shopId;
    private Integer cashierDeskId;
    private String orderNo;
    private String dateTime;
    private Integer orderType;
    private String orderTypeStr;
    private String price;
    private String realPrice;
    private String discountPrice;

    public String getRealPrice() {
        return realPrice;
    }

    public void setRealPrice(String realPrice) {
        this.realPrice = realPrice;
    }

    public String getDiscountPrice() {
        return discountPrice;
    }

    public void setDiscountPrice(String discountPrice) {
        this.discountPrice = discountPrice;
    }

    public Integer getShopId() {
        return shopId;
    }

    public void setShopId(Integer shopId) {
        this.shopId = shopId;
    }

    public Integer getCashierDeskId() {
        return cashierDeskId;
    }

    public void setCashierDeskId(Integer cashierDeskId) {
        this.cashierDeskId = cashierDeskId;
    }

    public String getOrderNo() {
        return orderNo;
    }

    public void setOrderNo(String orderNo) {
        this.orderNo = orderNo;
    }

    public String getDateTime() {
        return dateTime;
    }

    public void setDateTime(String dateTime) {
        this.dateTime = dateTime;
    }

    public Integer getOrderType() {
        return orderType;
    }

    public void setOrderType(Integer orderType) {
        this.orderType = orderType;
    }

    public String getOrderTypeStr() {
        return orderTypeStr;
    }

    public void setOrderTypeStr(String orderTypeStr) {
        this.orderTypeStr = orderTypeStr;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }
}

File: app/src/main/java/com/yp/payment/model/OrderDetail.kt
package com.yp.payment.model

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

@Entity
class OrderDetail{
    @Id
    var id: Long = 0
    var shopId: Int? = null
    var cashierDeskId: Int? = null
    var orderNo: String? = null
    var dateTime: String? = null
    var orderType: Int? = null
    var orderTypeStr: String? = null
    var price: String? = null
    var realPrice: String? = null
    var discountPrice: String? = null

    /**
     * 是否采用离线支付
     */
    var isPayByOffline:Boolean = false

    /**
     * 卡号
     */
    var cardNo:String = ""
}